<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Howe Sound Debris Pathway Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="Combo_Box_Season">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>680</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Summer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Winter</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Start_Button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>680</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Combo_Box_Location">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>680</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Northern Howe Sound</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Central Howe Sound</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Southern Howe Sound</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>680</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Season:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>680</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Location:</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>711</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>50</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="WindRose_Layout"/>
   </widget>
   <widget class="QLabel" name="Windrose_Label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>290</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>370</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="WindRose_Daytime_Layout"/>
   </widget>
   <widget class="QLabel" name="Windrose_Daytime_Label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>620</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
